packages/wrangler/src/containers/containers.ts

```diff
@@ -27,16 +27,20 @@ export function deleteYargs(args: CommonYargsArgv) {
 	return args.positional("ID", {
 		describe: "id of the containers to delete",
 		type: "string",
+		demandOption: true,
 	});
 }
 
 export async function deleteCommand(
 	deleteArgs: StrictYargsOptionsToInterface<typeof deleteYargs>,
 	_config: Config
 ) {
-	if (!deleteArgs.ID) {
+	// API gateway has path restrictions so if someone provides a string that isn't ID shaped, we get a weird error instead of a 404
+	const uuidRegex =
+		/^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$/i;
+	if (!uuidRegex.test(deleteArgs.ID)) {
 		throw new Error(
-			"You must provide an ID. Use 'wrangler containers list` to view your containers."
+			"You must provide the container ID. Use 'wrangler containers list` to view your containers and corresponding IDs."
```

Comment:
```suggestion
			"You must provide the container ID. Use `wrangler containers list` to view your containers and corresponding IDs."
```
since the string is anyways being updated